apps/antalmanac/src/components/RightPane/SectionTable/PrereqTree.tsx

```diff
@@ -40,30 +46,37 @@ interface TreeProps {
 
 const Tree: FC<TreeProps> = (props) => {
```

Comment:
Not part of your diff, but I feel like `Tree` sucks as a name for this class. Maybe throw in a name change to `PrereqTreeDisplayNode` or something else if you can think of a better one.